pkgs/tools/misc/sanctity/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sanctity";
+  version = "0.7.3";
+
+  src = fetchGit {
+    url = "https://codeberg.org/papojari/sanctity.git";
+    ref = "refs/tags/v0.7.3";
+  };
```

Comment:
Please add sha256 hash?

Reply:
Not that hash (the one specifying the commit), but the one verifying the content. The role is similar to `cargoSha256` . 
An example of how to use fetchgit would be `pkgs/applications/audio/ardour/default.nix`

Reply:
Also I have noticed that, fetching from codeberg.org has a common pattern to use `fetchFromGitea` . so insead of using `fetchgit` directly use below instead.

```nix
  src = fetchFromGitea {
    domain = "codeberg.org";
    owner = "papojari";
    repo = pname;
    rev = "v${version}";
    sha256 = "sha256-1bgk4HxWdefhNFeEQrg9Jvj3aMvzFfIJl+eYK/AgX5g=";
  };
```